x-pack/test/examples/triggers_actions_ui_examples/alerts_table.ts

```diff
@@ -107,61 +107,13 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       );
     });
 
-    it('should open a flyout and paginate through the flyout', async () => {
```

Comment:
@PhilippeOberti 
Instead of deleting this test, can we commented out and create a ticket to bring it back by creating a cell that can open the fly-out like we do in o11y. I will feel better around it.

Reply:
is [this](https://github.com/elastic/kibana/pull/168048/commits/bc0b4081c9c462c1f397d269092c6863e8fe5f55) what you had in mind?